### ISSUE TITLE
Unable to use for a particular div in a page

### ISSUE THREAD

**Issue by aravindnet** [sroze/ngInfiniteScroll#246]
i am trrying to use this directive for particular div in ap age instead of window then its not working

**Comment by rasmuslp** [comment]
Have you tried specifying a container with the `infinite-scroll-container` attribute?

**Comment by aravindnet** [comment]
yeah. itried but infinite scroll function is keep getting called always

**Comment by mudassir0909** [comment]
https://github.com/sroze/ngInfiniteScroll/issues/258#issuecomment-188188837

**Comment by umair-khanzada** [comment]
@aravindnet: I am facing too that issue. 
after lot of googling didn't found any solution then i change my css and it's working for me.
i remove the css " height " property and then it's working for me.

**Comment by aravindnet** [comment]
cool..... could you paste your code snippet here

**Comment by umair-khanzada** [comment]
yes sure.

Html
`<div class="container-fluid your-class" infinite-scroll='yourFunction()' 
infinite-scroll-disabled='busy'>`

css before when i am facing issue.
`.your-class{
  background: #E6E7E8;
  height: 100vh;    //vh = view height
}`

css after.
`.your-class{
  background: #E6E7E8;
  //height: 100vh;      //vh = view height
}`

**Comment by aravindnet** [comment]
Thanks for sharing it

**Comment by umair-khanzada** [comment]
can it work's for you ?

**Comment by aravindnet** [comment]
we taken other plugin to implement infinite scroll in our app. i am not
working on this now

**Comment by umair-khanzada** [comment]
okie.
can you try it in a demo app ?
if not so please try it.

**Comment by rajibbrunel** [comment]
@aravindnet 
can you tell me which plugin you are currently working
